src/ui/public/styles/base.less

```diff
@@ -157,6 +157,16 @@ a {
       .button-group > :last-child {
         border-radius: 0;
       }
+
+      button.is-kbn-top-nav-button-disabled {
```

Comment:
I tried that first but it didn't apply the style. I had to add `button` to make the selector more specific.